src/appleseed.studio/main/main.cpp

```diff
@@ -60,6 +60,7 @@
 #include <cstdio>
 #include <cstdlib>
 #include <cstring>
+#include <clocale>
```

Comment:
We require that `#include` clauses are always sorted alphabetically. This helps preventing duplicate includes.